instrumentation/elastic-opentelemetry-instrumentation-openai/tests/test_chat_completions.py

```diff
@@ -872,6 +874,146 @@ def test_tools_with_capture_content_log_events(
     )
 
 
+@pytest.mark.integration
+@pytest.mark.parametrize(
+    "provider_str,model,response_model",
+    [
+        (
+            "openai_provider_chat_completions",
+            "gpt-4o-mini",
+            "gpt-4o-mini-2024-07-18",
+        ),
+    ],
+)
+def test_tools_with_capture_content_log_events_integration(
+    provider_str,
+    model,
+    response_model,
+    trace_exporter,
+    logs_exporter,
+    metrics_reader,
+    request,
+):
+    provider = request.getfixturevalue(provider_str)
+    client = provider.get_client()
+
+    # Redo the instrumentation dance to be affected by the environment variable
+    OpenAIInstrumentor().uninstrument()
+    with mock.patch.dict(
+        "os.environ", {"ELASTIC_OTEL_GENAI_CAPTURE_CONTENT": "true", "ELASTIC_OTEL_GENAI_EVENTS": "log"}
```

Comment:
I know this would be in a different PR, but I think we should rename this variable to be the same as upstream, right?

ELASTIC_OTEL_GENAI_CAPTURE_CONTENT -> OTEL_INSTRUMENTATION_GENAI_CAPTURE_MESSAGE_CONTENT

Reply:
also log are default in 1.28.0, so we should switch precedence

Reply:
Yep, Will take care of these on a later PR.